tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -199,6 +202,36 @@ where
 		}
 	});
 
+	if cfg!(not(feature = "production")) {
+		// mrenclave_updateScheduled
+		// params: sidechainBlockNumber, hex encoded mrenclave
+		let mrenclave_update_scheduled_name: &str = "mrenclave_updateScheduled";
```

Comment:
regarding the name, would `state_updateScheduledEnclave` sound better? To be resonant with the parachain extrinsic

Reply:
I thought `state` refers to shared state ( the one replicated across the nodes) so I decided to not use it.
I can change it to `state_*`, this call is present only in `dev` mode anyway.